jsonrpc/testsuite/block-with-txn-bodies.json

```diff
@@ -0,0 +1,39 @@
+{
+    "parentHash": "0x0100000000000000000000000000000000000000000000000000000000000000",
+    "sha3Uncles": "0x0200000000000000000000000000000000000000000000000000000000000000",
+    "miner": "0x0100000000000000000000000000000000000000",
+    "stateRoot": "0x0400000000000000000000000000000000000000000000000000000000000000",
+    "transactionsRoot": "0x0500000000000000000000000000000000000000000000000000000000000000",
+    "receiptsRoot": "0x0600000000000000000000000000000000000000000000000000000000000000",
+    "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+    "difficulty": "0xa",
+    "totalDifficulty": "0x0",
+    "size": "0x0",
+    "number": "0xb",
+    "gasLimit": "0xc",
+    "gasUsed": "0xd",
+    "timestamp": "0xe",
+    "extraData": "0x616263646566",
+    "mixHash": "0x0700000000000000000000000000000000000000000000000000000000000000",
+    "nonce": "0x0a00000000000000",
+    "hash": "0x0800000000000000000000000000000000000000000000000000000000000000",
+    "transactions": [
+        {
+            "nonce": "0x1",
+            "gasPrice": "0xa",
+            "gas": "0x64",
+            "to": "0x0000000000000000000000000000000000000000",
+            "value": "0x3e8",
+            "input": "0x0102",
+            "v": "0x1",
+            "r": "0x2",
+            "s": "0x3",
+            "hash": "0x0200000000000000000000000000000000000000000000000000000000000000",
+            "from": "0x0300000000000000000000000000000000000000",
+            "blockHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+            "blockNumber": "0x1",
+            "transactionIndex": "0x2"
+        }
+    ],
+    "uncles": null
+}
```

Comment:
would be great to add the new line if it doesn't break the tests

Reply:
Cannot do it from my vscode. It removes the line.